src/components/frame/v5/pages/VerifiedPage/partials/TableItem/TableItem.tsx

```diff
@@ -54,7 +68,9 @@ const TableItem: FC<TableItemProps> = ({ member, onDeleteClick, onChange }) => {
           <Icon name="verified" appearance={{ size: 'tiny' }} />
         </span>
       </div>
-      <div className="hidden sm:flex items-center">status</div>
+      <div className="hidden sm:flex items-center">
```

Comment:
I guess we shoudn't show this <div> when `userStatus`  is undefined

Reply:
I have to show this div empty because some users might not have status and table will break

![image](https://github.com/JoinColony/colonyCDapp/assets/76940796/5356a41d-bf62-46c6-99ce-03b182be50da)